data/net/minecraft/world/level/levelgen/blending/BlendingData.mapping

```diff
@@ -2,5 +2,12 @@ CLASS net/minecraft/world/level/levelgen/blending/BlendingData
 	METHOD <init> (ZLjava/util/Optional;)V
 		ARG 1 oldNoise
 		ARG 2 heights
+	METHOD getDensityColumn (Lnet/minecraft/world/level/chunk/ChunkAccess;III)[D
+		ARG 0 chunkAccess
+	METHOD isGround (Lnet/minecraft/world/level/chunk/ChunkAccess;Lnet/minecraft/core/BlockPos;)Z
+		ARG 0 chunkAccess
```

Comment:
```suggestion
		ARG 0 chunk
	METHOD isGround (Lnet/minecraft/world/level/chunk/ChunkAccess;Lnet/minecraft/core/BlockPos;)Z
		ARG 0 chunk
```
There's not really a reason these need to be typed as per the interface, and there's precedent for using `chunk`, for instance in [`ChunkGenerator`](https://github.com/ParchmentMC/Parchment/blob/versions/1.18.x/data/net/minecraft/world/level/chunk/ChunkGenerator.mapping) methods.